spec/system/visit_msa_component_edit_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Edit MSA Component Page', type: :system do
+  before(:each) do
+    login_user
+  end
+
+  let(:msa_component_1) { create(:msa_component) }
+  let(:team1) { NewTeamEvent.create(name: 'team 1').team }
```

Comment:
not a big fan of adding numbers when there's only one instance...

Reply:
I had multiple instances before. Thanks